application/src/sass/cms/_banner.scss

```diff
@@ -14,8 +14,8 @@ This serves the dual purposes of reminding publishers that they're looking at a
   padding: 5px 10px;
 }
 
-.cms-banner a {
-  color: inherit;
+.cms-banner a, button.link {
```

Comment:
We already have a `button.link` defined in `_button.scss` - should be able to use that?

Reply:
It has blue text though, which is why it needs this to override the blue.

Reply:
This was exactly the thing I was trying to do in the first place.  For some reason I imagined that the `.cms-banner` at the start of the line applied to all the things afterwards, and I couldn't figure out why this wasn't overriding the default `button.link` styles :man_facepalming: